src/Api/Endpoints/ProductEndpoints.cs

```diff
@@ -71,6 +72,7 @@ private async Task<IResult> UpdateProduct([FromBody] UpdateProductRequest reques
         }
         catch (ValidationException validationException)
         {
+            logger.LogError("{ErrorMessage} {Errors}", validationException.Message, validationException.Errors);
```

Comment:
Ditto (not error)

Reply:
Ditto: Nice question, it depends on how do we want to log it.

Reply:
Finally, I decided to switch to the Information way.